R/implicit_assignment_linter.R

```diff
@@ -56,9 +56,11 @@ implicit_assignment_linter <- function(except = c(
     //SYMBOL_FUNCTION_CALL"
   }
 
+  # The walrus operator `:=` is also `LEFT_ASSIGN`, but is not a relevant operator
+  # to be considered for the present linter.
   assignments <- c(
-    "LEFT_ASSIGN", # e.g. mean(x <- 1:4)
-    "RIGHT_ASSIGN" # e.g. mean(1:4 -> x)
+    "LEFT_ASSIGN[text() = '<-']", # e.g. mean(x <- 1:4)
```

Comment:
this will exclude `<<-` and `->>`, though, would `text() != ':='` be better?

Reply:
Good catch!

I forgot about super-assignments because there were no tests for them. I have made the necessary change and also added tests for the super-assignment operators.